dt: 0.1
human_control_law: CONST_ACC
human_kynematic_model: KEYPOINTS
human_n_dof: 3    # Number of degrees of freedom (i.e., number of joints if human_kynematic_model is KYN_CHAIN)
human_n_kpts: 18  # Number of keypoints (equal to human_n_dof if human_kynematic_model is KEYPOINTS)
human_noisy_model: False
human_noisy_measure: True
human_meas_variance: # Measurement noise on the position of each human keypoint (Hip: all keypoints have the same variance)
  x: 0.05
  y: 0.05
  z: 0.05
human_model_variance: # Model uncertainty for each human DoF
  pos: 5.0
  vel: 50.0
  acc: 50.0
human_init_variance: # [pos, vel, acc] for each human DoF
  pos: 10.0
  vel: 10.0
  acc: 100.0
robot_init_variance: # [pos, vel, acc] for each robot DoF
  pos: 0.01
  vel: 0.1
  acc: 1.0
human_Kp: 1.0
human_Kd: 1.0
human_K_repulse: 1.0
robot_control_law: TRAJ_FOLLOW
robot_n_dof: 6
alpha: 0.3
beta: 2.0
kappa: 0.1
skeleton_topic: "/zed/zed_node/body_trk/skeletons"
robot_js_topic: "/ur10/joint_states"                
predicted_hri_state_topic: "/predicted_hri_state"
predicted_hri_cov_topic: "/predicted_hri_variance"
human_meas_topic: "/human_meas_pos"
human_filt_pos_topic: "/human_filt_pos"
human_filt_vel_topic: "/human_filt_vel"
camera_frame: "zed_camera_link"
world_frame: "world"
hz: 100 # remove
num_steps: 5 # check when is 0 (uncaught exception)
dump_to_file: False
plot_covariance: False
TF_world_camera: [0.100575, -0.9304, 2.31042, 0.180663, 0.516604, 0.119341, 0.828395] # [x y z qx qy qz qw]
u_min_human: -100
u_max_human: 100
a_min_human: -50
a_max_human: 50
v_min_human: -5
v_max_human: 5